### PR TITLE
Clean up Reddit and YouTube links

### PR DESCRIPTION
**Pull request by discomrade**

### CODE REVIEW

File: us_hate_crimes.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ _Definition: a prejudice-motivated crime which occurs when a perpetrator targets
...
-- On July 14th, 2017, alt-right activist, anti-feminist, and former intern of Milo Yiannapoulous, Lane Davis, [murdered his father for calling him a nazi](https://www.thedailybeast.com/youtube-trumpkin-and-former-milo-intern-kills-his-own-dad-for-calling-him-a-nazi).
+- On July 14th, 2017, alt-right activist, anti-feminist, and former intern of Milo Yiannapoulous, Lane Davis, [murdered his father for calling him a nazi](https://www.thedailybeast.coyoutube-trumpkin-and-former-milo-intern-kills-his-own-dad-for-calling-him-a-nazi).
```
Comment (dessalines) [inline us_hate_crimes.md:35]: Check this one out. What was the regex / replacer you used?

Comment (discomrade) [reply us_hate_crimes.md:35]: Well spotted, I forgot to escape the `.` in a rushed `sed -i "s/m.youtube/youtube/g" *.md`

I can't spot any others impacted by that one overreaching so I've just force-pushed a correction.

Comment (dessalines) [reply us_hate_crimes.md]: Sweet, thx.

**Review by dessalines — APPROVED** [review]
LGTM, thx!

**Pull request merged**